src/VisualStudio/Setup.Next/project.json

```diff
@@ -1,7 +1,10 @@
 {
   "dependencies": {
     "StreamJsonRpc": "0.14.3-alpha",
-    "Newtonsoft.Json": "6.0.6"
+    "Newtonsoft.Json": "6.0.6",
+    "Microsoft.VisualStudio.Threading": "14.1.131",
+    "Microsoft.VisualStudio.Validation": "14.1.111",
+    "RoslynDependencies.Microsoft.Build.Desktop": "14.3.25407"
```

Comment:
We should also replace the current Roslyn.Microsoft.Build dependency in all our project.json files with this new nuget package.